pkg/server/runner.go

```diff
@@ -428,13 +429,37 @@ func (r *Server) Flags(cmd *cli.Command) {
 }
 
 func (r *Server) Before(cmd *cli.Command) {
+	pb := cmd.Before
+	cmd.Before = func(c *cli.Context) error {
+		l := log.GetLogger()
+
+		// Sink the output of standard logger to util logger.
+		stdlog.SetOutput(l)
+
+		// Turn on the logrus logger
+		// and sink the output to util logger.
+		logrus.SetLevel(logrus.TraceLevel)
```

Comment:
Why do we set this level?

Reply:
`logrus logger -> [sink to] -> util logger`: turn on the lowest level of the logrus logger, so we can limit the level output of "util logger".